go/enclave/enclave.go

```diff
@@ -574,7 +574,7 @@ func (e *enclaveImpl) CreateBatch() common.SystemError {
 
 	callStart := time.Now()
 	defer func() {
-		e.logger.Info(fmt.Sprintf("CreateBatch call ended - start = %s duration %s", callStart.String(), time.Since(callStart).String()))
+		e.logger.Info("CreateBatch call ended", "duration", time.Since(callStart))
```

Comment:
Should we convert that Duration object to millis or something so they can be compared or does datadog have some magic to parse time units?